### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by BZOzyurt**

### CODE REVIEW

**Review by mayuces — APPROVED** [review]
Approved. But there is no GH Page Link

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleDataLoading = async (request: Promise<Good[]>) => {
```
Comment (VinnieJ-2k20) [inline src/App.tsx:10]: Apply useCallback here.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Looks good, just optimise the code using useCallback.

**Comment by VinnieJ-2k20** [comment]
And add DEMO link

**Pull request closed without merging**